config/env_spec_request.go

```diff
@@ -165,14 +161,34 @@ func (e *EnvironmentSpecRequest) getClaimValue(claim JWTClaim) string {
 	return ""
 }
 
+// JWTAuthentications returns a list of JWTAuthentication specific to the request.
+func (e *EnvironmentSpecRequest) JWTAuthentications() []*JWTAuthentication {
+	var auths []*JWTAuthentication
+	for _, v := range e.operation.jwtAuthentications {
```

Comment:
same Q as above

Reply:
Unlike previous one, this one is called in the context of a request, so it has narrower scope. If the op has a non-zero map, you can see it directly returns here. Otherwise, it collects what's in the api. There is no merging.

Reply:
Yup. I missed the context change. Thanks.